src/main/java/skajihara/projectX/MainContents/Home/service/AccountService.java

```diff
@@ -0,0 +1,22 @@
+package skajihara.projectX.MainContents.Home.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import skajihara.projectX.MainContents.Home.entity.Account;
+import skajihara.projectX.MainContents.Home.exception.AccountException;
+import skajihara.projectX.MainContents.Home.repository.AccountRepository;
+
+@Service
+public class AccountService {
+
+    @Autowired
+    private AccountRepository accountRepository;
+
+    public Account selectAccount(String id) {
+        Account account = accountRepository.selectAccount(id);
+        if (account == null) {
+            throw new AccountException("Account id: " + id + " is not found");
```

Comment:
同様です。
```suggestion
            throw new NotFoundException("Account id: " + id + " is not found");
```
https://spring.pleiades.io/specifications/platform/8/apidocs/javax/ws/rs/notfoundexception#NotFoundException-java.lang.String-

Reply:
修正しました。
org.springframework.data.crossstore.ChangeSetPersister.NotFoundExceptionを使用したのですが、こちらはSpring Dataで用意されており、拡張せずに直接オーバーライドができなかったため、そのままHTTP ERROR 404エラーページを表示しております。

Reply:
`javax.ws.rs.NotFoundException` とかでできないんだっけ。

Reply:
NotFoundExceptionクラスを作成し、コードを修正しました。